### PR TITLE
ACM-9830--carefully changing, eliminating, or reducing long ways of saying simple things

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/cluster_lifecycle/import_gui.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ After you install {mce}, you are ready to import a cluster to manage. Continue r
 [#import-gui-prereqs]
 == Prerequisites
 
-* A deployed hub cluster on a supported {ocp-short} version.
+* A deployed hub cluster. If you are importing bare metal clusters, the hub cluster must be installed on a supported {ocp} version.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/import_gui.adoc:15]: I think here I cut this down and merged in another PR but that diff is not here, I had some conflicts. Basically a shorter way to say what was there before.

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repository
-        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
```
Comment (swopebe) [inline .github/workflows/linting.yml:13]: I am not sure what this is about.

File: observability/customize_observability.adoc [hunk 3]
```diff
@@ -127,6 +127,8 @@ data:
 <2> *Optional:* Enter only one value for the `expr` and `record` parameter pair to define the query expression. The metrics are collected as the name that is defined in the `record` parameter from your managed cluster. The metric value returned are the results after you run the query expression.
 +
 You can use either one or both of the sections. For user workload metrics, see the _Adding user workload metrics_ section.
++
```
Comment (swopebe) [inline observability/customize_observability.adoc:130]: I didn't make this change.

File: .github/workflows/linting.yml (hunk 2 above)
Comment (dockerymick) [reply .github/workflows/linting.yml:13]: This is related to the PR Dale created related to the linter: https://github.com/stolostron/rhacm-docs/pull/6437

File: observability/customize_observability.adoc (hunk 3 above)
Comment (dockerymick) [reply observability/customize_observability.adoc:130]: I made this change in my PR that was merged Friday: https://github.com/stolostron/rhacm-docs/pull/6652/files

**Review by dockerymick — APPROVED** [review]

**Pull request merged**